test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetVerifyCommandTest.cs

```diff
@@ -92,6 +96,51 @@ public void VerifyCommand_WithAuthorSignedPackage_FailsGracefully()
             }
         }
 
+        [Theory]
+        [InlineData("", RevocationMode.Online)]
+        [InlineData(null, RevocationMode.Online)]
+        [InlineData("online", RevocationMode.Online)]
+        [InlineData("offline", RevocationMode.Offline)]
```

Comment:
Add a scenario that varies casing, since you [explicitly allow case-insensitive enum parsing](https://github.com/NuGet/NuGet.Client/pull/2391/files#diff-f83497230965c941a0bf5e2d2ed13deeR325).

Is the setting name `revocationMode` case-sensitive?